packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -399,22 +408,26 @@ function WorklistDataGrid(props) {
       id: rowIdsToSelect.length === 1
         ? 'Terra.worklist-data-grid.row-selection-template'
         : 'Terra.worklist-data-grid.row-selection-selected-rows-range',
-    }, { row: rangeInfo.anchorRow, endRow: newEndRowIndex })}`;
+    }, { row: startOfRange, endRow: newEndOfRange })}`;
 
     setAriaLiveMessage(ariaMessage);
     onRowSelect(rowIdsToSelect, rowIdsToUnselect);
```

Comment:
I still think we should just have one array for our API.  As a consumer, I would not want to have to deal with two separate arrays to manage my state.  I don't think we should introduce the two array approach to our consumers.

Reply:
Changed